docs/internals/controls-package.md

```diff
@@ -9,20 +9,286 @@ description: "The Control Package implements feedback control on an endpoint."
 
 ## Introduction
 
-The control package implements feedback control on an endpoint (plant).
+The easiest way to utilize the controls package is by configuring the `control_parameters` attribute for an [encoded motor](/docs/components/motor/gpio/encoded-motor/) and/or a [sensor-controlled base](/docs/components/base/sensor-controlled/). However, if you want to change or customize the control loops on these components, or you want to add a control loop to a different component, you can use the controls package to [build your own PID control loop](/internals/controls-package/#creating-and-using-a-pid-control-loop).
```

Comment:
not the most info here, let me know if you have any suggestions on expanding on this